.gitlab-ci.yml

```diff
@@ -105,6 +105,7 @@ include:
       - if: '$CI_PROJECT_NAME == "mender-server"'
         when: always
       - when: never
+  - remote: 'https://raw.githubusercontent.com/mendersoftware/mendertesting/master/qa-common/send-results-to-mantra.yml'
```

Comment:
Same as I commented in [the other PR,](https://github.com/mendersoftware/integration/pull/2809#discussion_r2509168553) I am positive that the `mender-server` maintainers will appreciate consistency in the include style here :wink: